portfolio/src/main/client/package-lock.json

```diff
@@ -0,0 +1,13246 @@
+{
```

Comment:
I've heard that it's a best practice to commit `package-lock.json` to make sure that every time I download my code, it will install the exact same package versions when I run `npm install`, which will avoid things breaking unpredictably. Do you still think I should remove it?

Reply:
Interesting. We don't have to do much like this internally. I'll do some digging, but this seems like solid reasoning. @jessicaslaughter WDYT?

Reply:
Yeah I think it's fine to keep it here, just because I don't know enough about this to argue for adding it to .gitignore (there are people saying it's fine to ignore here): https://stackoverflow.com/questions/44206782/do-i-commit-the-package-lock-json-file-created-by-npm-5)